### ISSUE TITLE
InvertEBase

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#4238]
@sharpphyl do you guys also publish to InvertEBase? https://invertebase.org

**Comment by Jegelewicz** [comment]
OOPS - found y'all there! Did you have to request publication or are they just picking up stuff from the IPT?

![image](https://user-images.githubusercontent.com/5725767/148792335-2ea420dd-f89f-4dfc-a749-dfe2c8757651.png)

**Comment by sharpphyl** [comment]
@Jegelewicz     We did not take any action to become part of InvertEBase.  I stumbled across our inclusion a number of years ago and was, of course, delighted to see we had somehow been added.  They have almost doubled the number of collections in recent years.  They are probably picking up the data from the IPT.  I see that our data was updated 4 Jan 2022 and we took no action to make this happen.

If you want to know more (or suggest other collections for inclusion), I would contact Petra Sierwald at psierwald@fieldmuseum.org or Rudiger Bieler at rbieler@fieldmuseum.org.  We contacted Petra when we needed to update our collection description and contact information.  When we had a question a few years ago about the frequency of their uploads, the answer ultimately came from Ed Gilbert at egbiodiversity@gmail.com.

You'll note from their "about" page that 

> In 2020, the InvertEBase portal was greatly expanded as a result of merging the Smithsonian Tropical Research Institute’s Marine Life of Panama Symbiota portal, and the addition of two new TCNs funded by NSF that will use and further develop the portal.

This has resulted in our data oddly appearing directly under the STRI's portal image.  I've run across it once or twice on the internet, but it hasn't created any issues and it's obvious that the records are from DMNS.



<img width="1311" alt="Screen Shot 2022-01-13 at 6 37 22 AM" src="https://user-images.githubusercontent.com/15368365/149340618-588a2b3d-b0f9-419f-8347-9cdc5f8beea5.png">
